toolkit/scripts/build_tag.mk

```diff
@@ -34,6 +33,15 @@ RELEASE_VERSION := $(RELEASE_VERSION)
 # Image tag - empty by default. Does not apply to the initrd.
 IMAGE_TAG          ?=
 
+# Basis of the distro macros. The abreveation of the distro name and the major version are encoded into the go tools so
+# that they can be used to create the distro specific macro '<DIST_NAME_ABRV> <DIST_MAJOR_ID>'.
+DIST_NAME_MACRO     ?= azl
+dist_major_macro    := $(shell echo $(RELEASE_MAJOR_ID) | cut -d'.' -f1)
+distro_macro        := $(DIST_NAME_MACRO) $(dist_major_macro)
+
+# Dist tag will be of the form '.<ABRV><MAJOR_ID>', unless overridden by the user.
+DIST_TAG            ?= .$(DIST_NAME_MACRO)$(DIST_MAJOR_ID)
```

Comment:
Consistency nit-pick: you use "ABRV", "MACRO" interchangeably making it a bit confusing to understand what's going on. Also, I think the comment above `DIST_TAG` is not adding any more information, since it's only visible here right above the actual value for `DIST_TAG`. Saying `<ABRV><MAJOR_ID>` in the comments while passing `$(DIST_NAME_MACRO)$(DIST_MAJOR_ID)` in reality adds to the confusion in my opinion.

What do you think about this:
```suggestion
##help:var:DIST_NAME_ABRV:<dist_name_abrv>=Basis of the distro macros. The abbreviation of the distro name and the major version are encoded into the go tools so that they can be used to create the distro specific macro '<DIST_NAME_ABRV> <dist_major_version_number>', where 'dist_major_version_number' is the major version number extracted from '<RELEASE_MAJOR_ID>'.
DIST_NAME_ABRV            ?= azl
dist_major_version_number := $(shell echo $(RELEASE_MAJOR_ID) | cut -d'.' -f1)
dist_version_macro        := $(DIST_NAME_ABRV) $(dist_major_version_number)

##help:var:DIST_TAG:<dist_tag>=Distribution tag, defines the "dist" macro used by the specs. Default: con.
DIST_TAG            ?= .$(DIST_NAME_ABRV)$(DIST_MAJOR_ID)
```

**NOTE**: I've re-named `distro_macro` to `dist_version_macro` to avoid confusion with `DIST_TAG` and use "dist" consistently.

Reply:
I was changing a few bits here, merged your suggestions in.